.github/workflows/ci.yaml

```diff
@@ -35,18 +35,16 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - name: Install and test NPM packages
+    - name: Run unit tests
```

Comment:
This is combining both Go and TS steps the same job. Since there's a matrix across language-specific dimensions such as node-version, this is going to be unnecessary duplication of work.

I think we should keep job separate per langauge.